docs/docs/getting-started/first-steps.md

```diff
@@ -128,6 +128,12 @@ If you don't have a cloud subscription, check out [MiniConstellation](first-step
 
 4. Initialize the cluster
 
+    :::caution
+
+    Initialization on **Azure** might be slow and might take up to 60 minutes to initialize all K8s nodes. We are already working with Microsoft to resolve the issue. The stability of already initialized clusters is not affected.
```

Comment:
```suggestion
    Initialization on **Azure** might be slow and might take up to 60 minutes to initialize all Kubernetes nodes. We're already working with Microsoft to resolve the issue. The stability of already initialized clusters isn't affected.
```
Imo we shouldn't use the 'k8s' short form in our docs. And is there any reason to ignore the vale language recommendations?

Reply:
Agreed. Shouldn't use K8s.

Reply:
Also agree to that we should follow Vale recommendations except for the "we", which is OK in most cases.

Reply:
I've added #820 to fix those.